### PR TITLE
Pact Contract Test

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: test/contract/logs/pact.log [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "json_class": "Pact::SomethingLike",
+        "contents": "MINE123"
+      }
+    },
+    "min": 1
+  }
+}
+I, [2020-04-19T14:23:48.927773 #279]  INFO -- : Verifying - interactions matched
+I, [2020-04-19T14:23:48.935182 #279]  INFO -- : Cleared interactions
+I, [2020-04-19T14:23:48.943478 #279]  INFO -- : Writing pact for ffc-demo-payment-service to /home/node/test/contract/pacts/ffc-demo-payment-web-ffc-demo-payment-service.json
```
Comment (jaucourt) [inline test/contract/logs/pact.log:58]: In the walkthrough, the logs and pacts directories are added to the .gitignore file

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-payment-web",
   "description": "Digital service mock to  view scheduled payments for financial aid in the event a property subsides into mine shaft.",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (jaucourt) [inline package.json:4]: I wonder if we should increment the minor rather than the revision number?

File: docker-compose.test.yaml [hunk 3]
```diff
@@ -17,4 +17,4 @@ services:
       OKTA_CLIENT_SECRET: placeholdersecret
       OKTA_AUTH_SERVER_ID: placeholderauthserverid
     volumes:
-      - ./test-output/:/home/node/test-output/
+      - ./test-output/:/home/node/test-output/
```
Comment (johnwatson484) [inline docker-compose.test.yaml:20]: Some sneaky whitespace added here

File: test/contract/ffc-demo-payment-service.pact.test.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  afterEach(async () => {
+    await provider.verify()
+  })
+
+  afterAll(async () => {
+    await provider.finalize()
+  })
+})
+
+/* const getSampleSchedules = () => [
```
Comment (johnwatson484) [inline test/contract/ffc-demo-payment-service.pact.test.js:106]: Commented out code needs removing

File: jest.setup.js [hunk 5]
```diff
@@ -1 +1 @@
-jest.setTimeout(10000)
+jest.setTimeout(30000)
```
Comment (spattisson) [inline jest.setup.js:1]: Do we need to leave this as 30000?

File: test/contract/ffc-demo-payment-service.test.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    expect(response[0]).toEqual(expect.objectContaining({
+      claimId: 'MINE123',
+      paymentDate: '2015-08-06T16:53:10+01:00'
+    }))
+  })
+
+  test('Get schedule by claim id returns schedule info', async () => {
+    await provider.addInteraction({
+      state: 'schedule exists',
+      uponReceiving: 'get schedule',
```
Comment (spattisson) [inline test/contract/ffc-demo-payment-service.test.js:57]:
```suggestion
      uponReceiving: 'get schedule by claim id',
```

File: test/contract/ffc-demo-payment-service.test.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    expect(schedule).toEqual(expect.objectContaining({
+      claimId: 'MINE001',
+      paymentDate: '2015-08-06T16:53:10+01:00'
+    }))
+  })
+
+  test('Get schedule by claim id returns 404 where claim id isn\'t found', async () => {
+    await provider.addInteraction({
+      state: 'schedule doesn\'t exist',
+      uponReceiving: 'get schedule',
```
Comment (spattisson) [inline test/contract/ffc-demo-payment-service.test.js:84]:
```suggestion
      uponReceiving: 'schedule by claim id',
```

File: jest.setup.js (hunk 5 above)
Comment (johnwatson484) [reply jest.setup.js:1]: I don't think this ended up being the root cause of the issue we were facing so can probably be restored.

**Pull request merged**